signer/signer.go

```diff
@@ -32,15 +32,25 @@ type Subject struct {
 	Names []csr.Name `json:"names"`
 }
 
+// Extension represents a raw extension to be included in the certificate.  The
+// "value" field must be Base64 encoded, using the URLEncoding variant (i.e.,
```

Comment:
It looks like CFSSL uses `StdEncoding` throughout, so it would be more consistent here to use that.

Alternately, we could use hex encoding, which might make the DER structure of extensions slightly more accessible if you recognize them. I could flip a coin either way, but I think URLEncoding is wrong.

Also, can you specify the semantics of what happens if an extension is present in both the CSR and in the SignRequest? Does one take precedence? Do they both get appended?